src/DataType.purs

```diff
@@ -196,4 +197,8 @@ dataTypes = L.fromFoldable
    , dataType "Marker"
         [ "Arrowhead" × 0
         ]
+   , dataType "TextFragment"
+        [ cText × 1 -- Str (str)
+        , cLink × 2 --  Val v, Str (url)
```

Comment:
I’ve changes this from `Val` to `any`, which is closer to the type it would have in some (imaginary) type system (idea of these comments being to document the “user-level” typing for the constructors).